app/Http/Controllers/RegisterController.php

```diff
@@ -45,7 +46,14 @@ public function register(Request $request): InertiaResponse
     {
         try {
             $payload = $this->getValidatedRegistrationPayload($request->json()->all());
-            $this->submitRegistrationPayload($payload);
+            $user = $this->submitRegistrationPayload($payload);
+
+            if (!Auth::attempt(['email' => $user->email, 'password' => $payload['password']])) {
```

Comment:
This line will authenticate a user. Is the point to auto login the user after registering?

Reply:
yes